src/runtime/aot_executor/aot_executor_factory.cc

```diff
@@ -52,6 +52,11 @@ PackedFunc AotExecutorFactory::GetFunction(
       }
       *rv = this->ExecutorCreate(devices);
     });
+  } else if (name == "module_list") {
```

Comment:
to follow convention of other functions here, shall we say list_module_names?

Reply:
Done.